backend/audit/etl.py

```diff
@@ -58,25 +58,19 @@ def load_findings(self):
         for entry in findings_uniform_guidance_entries:
             findings = entry["findings"]
             finding = Finding(
+                award_seq_number=entry["award_index"],
```

Comment:
There is an open question for Matt on #1175 that might impact award_index type. Once he provides feedback, I will update you @pluribus-tony. Otherwise, this PR looks good to me.

Reply:
@pluribus-tony it was not me (I was offline and just got back). I was going to get back to you with some clarification about my previous comment first. If the field "award_index" remains a string, I would suggest to rename award_seq_number something like award_key instead. However, if "award_index" is changed to a number then "award_seq_number" does not need be renamed.